Assets/BossRoom/Scripts/Editor/BakingMenu.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.Rendering;
+using UnityEngine.XR;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Scripts.Editor
+{
+    /// <summary>
+    /// This is a script that creates a menu for baking lights (and changing other lighting features) for Boss Room.
+    /// </summary>
+    public class BakingMenu : MonoBehaviour
```

Comment:
Oh and just realized this class doesn't need to inherit from MonoBehaviour. Can just be marked as an abstract class.

Reply:
I changed it to be just `abstract class BakingMenu`, is that what you meant? :0